includes/class-amp-response-headers.php

```diff
@@ -75,7 +72,7 @@ public static function send_header( $name, $value, $args = array() ) {
 	public static function send_server_timing( $name, $duration = null, $description = null ) {
 		$value = $name;
 		if ( isset( $description ) ) {
-			$value .= sprintf( ';desc=%s', wp_json_encode( $description ) );
+			$value .= sprintf( ';desc="%s"', str_replace( array( '\\', '"' ), '', substr( $description, 0, 100 ) ) );
```

Comment:
Replacing `_` with space would have a more elegant result for class names.

Reply:
This logic here is specifically for ensuring the well-formedness of the `Server-Timing` header. If we want to use spaces instead of underscores then we should do that at:

https://github.com/Automattic/amp-wp/blob/9dde3226a5998cab2ba12983d63f0fa8111fb338/includes/templates/class-amp-content-sanitizer.php#L96